chore: improve form story env to see form status during development

connoratrug opened the pull request:
### What are the main changes you did
- setup the Form story to see what is happening to the form state ( values and error) 

### How to test
- explain here what to do to test this (or point to unit tests)

### Checklist
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

connoratrug committed:
chore: improve form story env to see form status during development

chinook25 commented on apps/tailwind-components/pages/Form.story.vue:
```diff
@@ -36,37 +37,66 @@ function refetch() {
+    ></FormFields>
+
+    <div class="basis-1/2">
+      <div>Demo controles:</div>
```
Do you mean controls, or checks?

chinook25 reviewed (approved):
Seems good to me

connoratrug committed:
update label

connoratrug committed:
update test

connoratrug committed:
chore: bump playwright version and use same every where

move tailwind e2e into tailwind project
remove 'old' vue test from tailwind project

connoratrug committed:
cleanup pr
